package.json

```diff
@@ -139,7 +139,7 @@
 						"default": [
 							{
 								"key": " ",
-								"name": "Commands...",
+								"name": "Commands",
```

Comment:
this is not a typo. It is intentional because this is not a submenu. Tell me if you don't like it.

Reply:
Makes sense